junit5-extension/src/test/java/io/kroxylicious/testing/kafka/junit5ext/InstanceFieldExtensionTest.java

```diff
@@ -8,24 +8,94 @@
 import java.util.concurrent.ExecutionException;
 
 import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.MockConsumer;
+import org.apache.kafka.clients.consumer.OffsetResetStrategy;
+import org.apache.kafka.clients.producer.Producer;
+import org.junit.jupiter.api.Order;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 
+import com.example.StubAnnotation;
+
 import io.kroxylicious.testing.kafka.api.KafkaCluster;
 import io.kroxylicious.testing.kafka.common.BrokerCluster;
 import io.kroxylicious.testing.kafka.invm.InVMKafkaCluster;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
 
 @ExtendWith(KafkaClusterExtension.class)
-public class InstanceFieldExtensionTest extends AbstractExtensionTest {
+class InstanceFieldExtensionTest extends AbstractExtensionTest {
 
     @BrokerCluster(numBrokers = 1)
     KafkaCluster instanceCluster;
 
+    @BrokerCluster(numBrokers = 1)
+    @Name("kafkaCLuster")
```

Comment:
Funny capitalisation

Reply:
fixed in 478820f0cee6e7fc36c98987032076ff00c4e258